src/components/DeleteConfirmationDialog.js

```diff
@@ -8,8 +8,7 @@ import Dialog, {
 } from 'material-ui/Dialog';
 import { Button } from 'material-ui';
 import { connect } from 'react-redux';
-import { deleteAsset } from '../redux/actions/assetRegisterApi';
-import { handleConfirmDelete } from '../redux/actions/deleteConfirmation';
+import { deleteAsset, handleConfirmDelete } from '../redux/actions';
```

Comment:
I'm more of the opinion that having actions/index.js was a mistake since it is just a whole load of DRY violations. I don't think this should be changed for this story but I've been trying to import actions directly from their modules.